docs/running.rst

```diff
@@ -142,6 +160,18 @@ Options:
     Solang will not give a warning about this problem.
 
 
+
+Starting a new project
+______________________________
+
+
+  solang new --target solana my_project
+
+Start a new Solang project with an example `flipper.sol <https://github.com/hyperledger/solang/blob/main/examples/solana/flipper.sol>`_ contract,
+next to an example configuration file.
+
+
+
```

Comment:
This Paragraph needs to explain the user in more detail what exactly a "Solang project" is (that it creates a new directory with a default config file is just the product of this command, but we also need to explain the purpose of these Projects)